supplementary_style_guide/style_guidelines/release-notes.adoc

```diff
@@ -63,108 +63,433 @@ Introduce each release note with a heading that summarizes the release note. Thi
++
+RHELDOCS-19009
+====
 
-* Plan to deprecate
-* Deprecate
-* Plan to remove
-* Remove
+====
+Service Binding Operator documentation removed::
+With this release, the documentation for the Service Binding Operator (SBO) has been removed because this Operator is no longer supported.
++
+TICKET-REFERENCE
+====
 
-When alternatives to or workarounds for deprecated features are available, clearly inform users about them.
+If your product presents deprecations and removals in a table, follow the guidance for deprecation tables.
 
-==== Referring to releases in deprecation and removal notices
-In general, avoid definitive statements about specific releases, release versions, or dates for deprecation or removal.
-When possible, use the phrase "is planned for a future release" because it accounts for the possibility of changes to the planned deprecation or removal timeline.
+* Remove the entry from the table when the release for that removal is no longer fully supported. Typically that means a removal is not carried on beyond n-2 versions of the product documentation.
 
-If you must be specific about a release, use provisional language to reflect the fluid nature of development plans and to acknowledge the potential for plans to change.
-For example, if you must cite a specific version, rather than stating "<x> will be deprecated in version 4.16", use "It is currently planned for <x> to be deprecated in version 4.16".
-Alternatively, if you must cite a deprecation or removal timeline and you want to avoid citing a specific release number, use a phrase such as "<x> is planned to be deprecated in the next release".
+.Example table of removed features 
+|===
+|Category |Feature or component |Version |Alternative action |More information
 
-==== Deprecation notice template
-[subs="+quotes"]
-----
-In __<product_name> <release>__, __<name_of_capability_or_feature>__ is deprecated and is planned to be removed in the __<deprecation_timeline>__. Red{nbsp}Hat will provide bug fixes and support for this feature during the current release lifecycle, but this feature will no longer receive enhancements and will be removed. As an alternative to __<name_of_capability_or_feature>__, you can use __<alternative_capability_or_feature_if_available>__ instead.
-----
+|Application management |Subscriptions |2.5 |Use GitOps for |See _<insert_link_to_GitOps>_ for more details.
+|===
 
-.Example deprecation notice doc text
-----
-In Red{nbsp}Hat OpenStack Platform (RHOSP) 14, the director graphical user interface is deprecated and is planned to be removed in a future release. Red{nbsp}Hat will provide bug fixes and support for this feature during the current release lifecycle, but this feature will no longer receive enhancements and will be removed.
-----
 
-==== Removal notice template
-[subs="+quotes"]
+=== Known issues
+Known issues are existing problems that customers should be aware of to mitigate them and avoid unnecessary reporting.
```

Comment:
This is awkward wording that I noticed particularly as I read the file. How about this instead?

> Known issues describe existing problems that customers should be aware of, so that they can mitigate them and avoid unnecessary reporting.

Reply:
```suggestion
Known issues describe existing problems that customers should be aware of, so that they can mitigate them and avoid unnecessary reporting.
```